jans-cedarling/cedarling/src/authz/entities/create.rs

```diff
@@ -50,20 +52,25 @@ impl<'a> EntityMetadata<'a> {
     pub fn create_entity(
         &'a self,
         schema: &'a CedarSchemaJson,
-        data: &'a TokenPayload,
+        token: &Token,
```

Comment:
why we use in `EntityMetadata::create_entity`  token: &Token,
but in the  `create_entity`  tkn_data: &TokenData ?

Why we can't use in both tkn_data: &TokenData?

Reply:
because the token needs to know what token it is to fetch the metadata since the metadata is set on a per-issuer basis (which can be different). Then see my replies above to see why I decided to tag the tokens like this.

Reply:
refactored in [18ed767](https://github.com/JanssenProject/jans/pull/10436/commits/18ed7673bc485b13e045d2159b04de12cfc8641a)

Reply:
Thanks,  
I just realized that, in the near future, we'll need to implement the creation of an entity for `TrustedIssuer` (from the policy store). With the current approach, this implementation is not feasible.  
We may need to use a structure that implements a specific `trait` or one that is data-agnostic. In other words, it should be able to handle more than just token data.  

Apologies for bringing this up so late. This issue can be addressed in a separate PR.

Reply:
that's fine. let's find a better way to do this in the future.